### ISSUE TITLE
Custom image Inference

### ISSUE THREAD

**Issue by kdh-awraw1019** [whai362/pan_pp.pytorch#91]
I want to inference about a custom image.

this image is not in 'Prepare datasets' ex) ic17, ic15, ...

How can I inference a custom image?

How should I change it here? ▼▼
## ./config/pan_pp/pan_pp_r18_ic15_joint_train.py
data = dict(
    batch_size=16,
    train=dict(
        type='PAN_PP_JointTrain',
        split='train',
        is_transform=True,
        img_size=736,
        short_size=736,
        kernel_scale=0.5,
        read_type='pil',
        with_rec=True
    ),
    test=dict(
        type='PAN_PP_IC15',
        split='test',
        short_size=720,
        read_type='pil',
        with_rec=True
    )
)

**Comment by EMRAN-SALEH-CORSEARCH** [comment]
Yeah, Same here. I want ro do inference of PAN++ on custom image. I do not see that in the repo.

**Comment by Devin521314** [comment]
How  to predict a new image using the training weight?
